py/desispec/qa/qa_quicklook.py

```diff
@@ -103,15 +103,15 @@ def get_frame(filetype,night,expid,camera,specdir):
 
     #- Create necessary input for desispec.frame
     frame = fits.open(framefile)
-    wave = frame[3].data
+    wave = frame[4].data
```

Comment:
This PR just highlighted that you are reading HDUs by number rather than name, and this change from 3->4 implies that QL will work for some version of the data but not others.  It is slightly mixing topics in this PR, but please update this code to read by EXTNAME instead of numbers.  e.g.
```
wave = frame['WAVE'].data
flux = frame['FLUX'].data
...
```
See the EXTNAME keyword of each HDU for what the names should be.  This allows us to add or reorder HDUs and still have code like this work.